lib/cli/src/commands/login.rs

```diff
@@ -0,0 +1,35 @@
+use clap::Parser;
+use dialoguer::Input;
+
+/// Subcommand for listing packages
+#[derive(Debug, Clone, Parser)]
+pub struct Login {
+    /// Registry to log into (default: wapm.io)
+    #[clap(name = "REGISTRY")]
+    pub registry: Option<String>,
```

Comment:
Should this be an argument? That way people could run `wasmer login --registry wapm.dev my-secret-token`. Having multiple optional positional arguments feels like it could be ambiguous.

It also lets us do something like this:

```rs
#[clap(long, env, default_value = "wapm.io")]
pub registry: String,
```

That way the only place that hard codes the default registry is that `clap` attribute.

Reply:
Yeah, that makes sense. 

I'm just looking at what some other tools do:

```console
$ cargo login -h
Save an api token from the registry locally. If token is not specified, it will be read from stdin.

Usage: cargo login [OPTIONS] [token]

Arguments:
  [token]

Options:
  -q, --quiet                Do not print cargo log messages
      --registry <REGISTRY>  Registry to use
      ...
  -h, --help                 Print help information

Run `cargo help login` for more detailed information.

$ npm login -h
Add a registry user account

Usage:
npm adduser

Options:
[--registry <registry>] [--scope <@scope>]
[--auth-type <legacy|web|sso|saml|oauth|webauthn>]

aliases: login, add-user

Run "npm help adduser" for more info

$ yarn login -h

  Usage: yarn [command] [flags]

  Stores registry username and email.

  Options:

    -v, --version                       output the version number
    ...
    --registry <url>                    override configuration registry
    ...
    -h, --help                          output usage information
  Visit https://yarnpkg.com/en/docs/cli/login for documentation about this command.
```

Reply:
Okay, then let's be consistent with other tools.